cms/static/js/spec/views/modals/edit_xblock_spec.js

```diff
@@ -134,7 +134,7 @@ describe('EditXBlockModal', function() {
         it('shows the correct title', function() {
             var requests = AjaxHelpers.requests(this);
             modal = showModal(requests, mockXModuleEditorHtml);
-            expect(modal.$('.modal-window-title').text()).toBe('Editing: Component');
+            expect(modal.$('.modal-window-title span.title').text()).toBe('Editing: Component');
```

Comment:
For Change #3 the goal is to move the edit icon further to the left when a window is 95%. 

Currently the issue is that both the title value and the title wrapper both have the class name .title, and that means both have a width 47% designation. With some css changes, the title value would benefit from its current title padding rules but would not have the 47% width set. 

(Tiniest very hacky css version of this would be to add class called title-value to the lower of the two current tittle class elements and make .title-value { width: initial !important} )

Currently: 
![image](https://user-images.githubusercontent.com/2023680/65064932-66b08900-d94f-11e9-9b4e-1d485d6018a5.png)


After Changes: 
![image](https://user-images.githubusercontent.com/2023680/65064723-04f01f00-d94f-11e9-9fe7-235caf07e1b9.png)